docs/integrations/destinations/s3-migrations.md

```diff
@@ -0,0 +1,202 @@
+# S3 Migration Guide
+
+# Upgrading to 1.0.0
+
+## Schema Changes
+
+This version introduces changes to the schema of data written to S3, which make it isomorphic to our [V2 certified database destinations](https://docs.airbyte.com/release_notes/upgrading_to_destinations_v2/#what-is-destinations-v2). The schema changes are as follows:
+
+| Old field name          | New field name           | Json Type          | Avro Type                                                 | Parquet Type                                 | Description                                                                                            |
+|-------------------------|--------------------------|--------------------|-----------------------------------------------------------|----------------------------------------------|--------------------------------------------------------------------------------------------------------|
+| `_airbyte_ab_id`        | `_airbyte_raw_id`        | string (UUID)      | `{ "type": "string", "logicalType": "uuid" }`             | `String`                                     | Airbyte-added unique identifier.                                                                       |
+| `_airbyte_emitted_at`   | `_airbyte_extracted_at`  | integer (epoch ms) | `{ "type": "string", "logicalType": "timestamp-millis" }` | `Int64(logicalType=Timestamp(Milliseconds))` | Time at which the data was loaded from the source.                                                     |
+| [NEW FIELD]             | `_airbyte_generation_id` | integer            | `{ "type": "long" }`                                      | `Int64`                                      | Monotonically-increasing refresh id, if applicable.                                                    |
```

Comment:
my initial glance missed the field renamings, b/c I went straight to the NEW FIELD rows. I think this would be a bit clearer at implying that the other rows also have important information

```suggestion
| [NONEXISTENT]             | `_airbyte_generation_id` | integer            | `{ "type": "long" }`                                      | `Int64`                                      | Monotonically-increasing refresh id, if applicable.                                                    |
```

Reply:
Done